helm/reana/values.yaml

```diff
@@ -178,6 +178,13 @@ traefik:
     enabled: true
     isDefaultClass: true
 
+dask:
+  enabled: false
+  cluster_max_memory_limit: "16Gi"
+  cluster_default_number_of_workers: 2
+  cluster_default_single_worker_memory: "2Gi"
+  cluster_max_single_worker_memory: "4Gi"
```

Comment:
The value of 4Gi does not correspond to what was advertised in the README file.  Please change to 8Gi.